src/lib/components/compositions/input-floating/floating-label.svelte

```diff
@@ -21,6 +21,7 @@
   export let type: 'date' | 'email' | 'number' | 'password' | 'phone' | 'text' = 'text';
   export let value = '';
 
+
```

Comment:
Not a huge deal, but if you manually added this blank lines, can you remove them. That way, the PR is not misleading regarding files changed.

Reply:
done